### PR TITLE
Fix some definition inaccuracies, typos

### PR DESCRIPTION
**Pull request by AdamISZ**
I'm not sure of being right on some of these, so please do let me know and I can change as needed.

The most unclear one is: I don't know why you called `is_xonly_t..` methods - they seem to be just booleans rather than methods that return booleans? I put 'type' but I don't have a strong opinion about it.

I might add some more as I keep reading.

### CODE REVIEW

File: bip-musig2.mediawiki [hunk 1]
```diff
@@ -247,7 +247,7 @@ Input:
 '''''GetSecondKey(pk<sub>1..u</sub>)''''':
 * For ''j = 1 .. u'':
 ** If ''pk<sub>j</sub> &ne; pk<sub>1</sub>'':
-*** Return ''pk<sub>j</sub>''
+*** Return ''bytes(pk<sub>j</sub>)''
```
Comment (jonasnick) [inline bip-musig2.mediawiki]: `pk<sub>j</sub>` is already a byte array

**Review by jonasnick — COMMENTED** [review]
I used "method" as a synonym for "scheme". "Types" could be misunderstood in a similar way. How about "mode"?

File: bip-musig2.mediawiki (hunk 1 above)
Comment (AdamISZ) [reply bip-musig2.mediawiki]: Right, my bad, I didn't chase it far up the stack enough :) will remove.

**Comment by AdamISZ** [comment]
> I used "method" as a synonym for "scheme". "Types" could be misunderstood in a similar way. How about "mode"?

sounds good.

File: bip-musig2.mediawiki (hunk 1 above)
Comment (AdamISZ) [reply bip-musig2.mediawiki]: Fixed in https://github.com/jonasnick/bips/pull/23/commits/789a909dd1ea0b4818e9e090b33066e44dd99a33

**Comment by AdamISZ** [comment]
OK done those two things.

So I'm not finding anything that looks wrong in the rest, so far, but: I have to admit I've completely failed to understand something about tweaks.

In `Sign` we have:

```
 d = gv⋅gaccv⋅gp⋅d' mod n
```
The extra factors account for sign flips for evenness, but I don't understand why tweak values aren't added in to the secret key? E.g. if I was doing a taproot tweak for `Q = P + tG` I would have to add `t` to my initial `sk`. Where does that come in? I even couldn't see it in the code, so I must be fundamentally misunderstanding something.

Is my mistake simply that the secret key tweak is implicit?

(Edit: I'm noticing that in `Sign`, the `tacc` is ignored in `Let (Q, gaccv, _, b, R, e) = GetSessionValues(session_ctx); fail if that fails`, which is maybe a more precise way of indicating what's puzzling me).

**Comment by jonasnick** [comment]
ACK, thanks.

The tweak value comes in at the `PartialSigAgg` stage: `e*g_v*tacc_v` is added to the sum of the partial signatures. If we did that in `Sign`, then every signer would add the tweak value leading to an incorrect result because it only needs to be added once.

**Review by real-or-random — COMMENTED** [review]
post-merge ACK

**Comment by AdamISZ** [comment]
> The tweak value comes in at the `PartialSigAgg` stage: `e*g_v*tacc_v` is added to the sum of the partial signatures. If we did that in `Sign`, then every signer would add the tweak value leading to an incorrect result because it only needs to be added once.

Oh of course, thanks. It seems I somehow managed to not read the final step of the procedure, the one that actually generates a signature :)

**Pull request merged**